src/style-library/components/pages/HeadingScreen.scss

```
@@ -0,0 +1,40 @@
+}
+
+// Preset names are full words ("Section Title"), not the short numeric labels the shared 4rem column
+// was sized for.
+.kadence-blocks-style-library__heading-screen .kadence-blocks-style-library__list-row-label {
+	width: 15rem;
+}
+
+// The live heading chip. `border-style` is left at `none` so a preset that sets only a border color or
+// width draws no frame — matching the block, where all three are needed before a border renders, and
+// keeping an unstyled preset from sprouting an edge the page would not have.
+//
```

Stylelint (17.14.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)